render.c

```diff
@@ -28,6 +28,11 @@ int render(struct mako_state *state, struct pool_buffer *buffer) {
 	int border_size = 2 * config->border_size;
 	int padding_size = 2 * config->padding;
 
+    int inner_margin = config->margin.top;
+    if (config->margin.bottom > config->margin.top) {
+        inner_margin = config->margin.bottom;
+    }
```

Comment:
Please use tabs instead of spaces

Reply:
I installed vim-sleuth so this will hopefully stop happening. :)

Reply:
I bet @emersion would merge an editorconfig if you wrote one

Reply:
I was considering that, so if you're both onboard I will definitely do so!

Reply:
Yeah, +1 to `.editorconfig`